package.json

```diff
@@ -27,7 +27,7 @@
     "test": "mocha",
     "lint": "gulp sasslint",
     "docs": "gulp sassdoc",
-    "commit": "yarn test; yarn lint; yarn docs"
+    "commit": "yarn test && yarn lint && yarn docs"
```

Comment:
This is needed to run `yarn commit` on Windows. I believe it should still work on Mac + Linux.